packages/examples/src/text.ts

```diff
@@ -0,0 +1,73 @@
+import { registerExamples } from './register';
+
+export const schema = {
```

Comment:
Maybe we can pick a different example, since an address is not typically entered in a single text field. Also reduce the `maxLength` to a size of 10 which is beneficial when playing around with the examples.